tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "suppressImplicitAnyIndexErrors": true,
+    "strictNullChecks": false,
+    "strict": true,
+    "target": "es6",
+    "moduleResolution": "node",
+    "module": "commonjs",
+    "outDir": "./dist/node",
+    "sourceMap": true,
```

Comment:
So we need source map for the node version? If so, would you please explain to me why?

Reply:
copied this from docs, will remove